src/omnisharp/utils.ts

```diff
@@ -19,6 +19,10 @@ export async function currentFileMembersAsTree(server: OmniSharpServer, request:
     return server.makeRequest<protocol.CurrentFileMembersAsTreeResponse>(protocol.Requests.CurrentFileMembersAsTree, request, token);
 }
 
+export async function blockStructure(server: OmniSharpServer, request: protocol.Request, token: vscode.CancellationToken) {
```

Comment:
In the protocol you defined on the OmniSharp side, this takes a `BlockStructureRequest`, which inherits from `SimpleFileRequest` rather than `Request`.